runelite-client/src/main/java/net/runelite/client/plugins/xpglobes/XpGlobe.java

```diff
@@ -36,6 +37,8 @@
 	private int goalXp;
 	private Instant time;
 	private double skillProgressRadius;
+	private BufferedImage skillIcon;
+	private int globeSize;
```

Comment:
I thought it would be better to keep it consistent with the rest of the file. I guess I can do use them for the others too.

Reply:
If the whole file uses getter and setter methods then you could leave it the way it is, but I do know Lombok is used in the project so it might better be changed to be consistent among the project as a whole.